src/routes/expocamera.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { Button, Text, StyleSheet, Image, View, TouchableOpacity } from 'react-native';
+import { Camera, Permissions } from 'expo';
+
+import colors from '../assets/colors.js'
+import metrics from '../themes/metrics.js'
+
+import CameraButtonSVG from '../components/CameraButtonSvg';
+import LargeText from '../components/LargeText';
+import Navbar from '../components/Navbar'
+
+class CameraExample extends React.Component {
+  state = {
+    hasCameraPermission: null,
+    type: Camera.Constants.Type.back,
+    image: null
+  };
+
+  async componentWillMount() {
+    const { status } = await Permissions.askAsync(Permissions.CAMERA);
+    this.setState({ hasCameraPermission: status === 'granted' });
```

Comment:
will this check happen a second time if we navigate away from and back to this page?